docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: '3.6'
+services:
+  postgres:
+    image: nhost/postgres:12-v0.0.6
+    restart: always
+    volumes:
+      - ./docker/db_data/db:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-secretpgpassword}
+    ports:
+      - '5432:5432'
+  graphql-engine:
+    image: hasura/graphql-engine:v2.1.0
```

Comment:
Let's use `v2.1.1` which is the latest and the one we use for customers in production.

Reply:
ok